.github/workflows/auto-release.yml

```diff
@@ -8,11 +8,8 @@ on:
       - production
 
 jobs:
-  publish:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: cloudposse/github-action-auto-release@v1
-        with:
-          prerelease: false
-          publish: true
-          config-name: auto-release.yml
+  auto:
+    uses: cloudposse/.github/.github/workflows/shared-auto-release.yml@main
+    with:
+      publish: true
```

Comment:
The functionality of being able to not publish a release on merge to main should be restored in the go repos. We often merge many different PRs before cutting a new release. It used to be:

https://github.com/cloudposse/terraform-provider-context/blob/main/.github/workflows/auto-release.yml#L23C20-L23C96

Reply:
Add `no-release` label to a PR that should not cut release automatically

Reply:
Now we have the logic here 

https://github.com/cloudposse/github-action-auto-release/blob/main/action.yml#L75

Reply:
As I mentioned on the other PR with these same changes, I didn't realize that logic had been moved upstream. It's confusing when you are saying `publish: true` when calling the workflow, but some other logic upstream is determining if it should be published.